### PR TITLE
Explore the getty

### PR DESCRIPTION
**Pull request by alyxrossetti**

### CODE REVIEW

File: gpi/frontend/web/src/ui/components/header-explore.vue [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            <span class="icon is-small">
+              <font-awesome-icon icon="caret-down" />
+            </span>
+          </button>
+        </div>
+        <div
+          id="dropdown-menu"
+          class="dropdown-menu"
+          role="menu">
+          <div class="dropdown-content">
```
Comment (rit) [inline gpi/frontend/web/src/ui/components/header-explore.vue]: @alyxrossetti Please wrap the `a` inside `div`. Use the `v-for` and `data` prop to render the list so it's more testable.

File: gpi/frontend/web/src/ui/components/header-explore.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="top-bar">
+    <div class="container">
+      <div
+        :class="{'is-active': isDropdownActive}"
+        class="dropdown is-hoverable">
```
Comment (rit) [inline gpi/frontend/web/src/ui/components/header-explore.vue]: `is-hoverable` makes the `toggle` feature to have unexpected behavior. When hovering over the button, the links appear. But after clicking on the button, the links are expected to disappear but they are not.

I think we can fix it by removing the hovering feature altogether.

File: gpi/frontend/web/src/ui/components/header-explore.vue (hunk 1 above)
Comment (alyxrossetti) [reply gpi/frontend/web/src/ui/components/header-explore.vue]: Done!

File: gpi/frontend/web/src/ui/components/header-explore.vue (hunk 2 above)
Comment (alyxrossetti) [reply gpi/frontend/web/src/ui/components/header-explore.vue]: @rit I think this is working as expected. There are 2 different functionalities: hover, and `toggle`. On desktop, you would probably just hover, but for mobile Kristen usually likes the click. If you click once the menu stays, if you click again it goes away...

Comment (alyxrossetti) [reply gpi/frontend/web/src/ui/components/header-explore.vue]: `is-hoverable` removed

**Comment by alyxrossetti** [comment]
conflicts resolved

**Comment by rit** [comment]
lgtm. Please merge it.

**Pull request merged**